packages/@dcl/inspector/src/components/EntityInspector/AnimatorInspector/utils.ts

```diff
@@ -22,14 +22,17 @@ export function isValidSpeed(speed: string | undefined): boolean {
 }
 
 export function mapAnimationGroupsToStates(animations: AnimationGroup[]): PBAnimationState[] {
-  return animations.map(($) => ({
-    clip: $.name,
-    playing: !!$.isPlaying,
-    weight: $.weight ?? 1,
-    speed: $.speedRatio ?? 1,
-    loop: $.loopAnimation ?? false,
-    shouldReset: false
-  }))
+  return animations.map(($) => {
+    const weight = isValidWeight($.weight.toString()) ? $.weight : 1
```

Comment:
the default in the validWeight is `0` but here is `1`

Reply:
Hey @gonpombo8, thanks for the review! 

we are using the default value `1` here to keep consistency with the `PBAnimationState` component.

updated the `isValidWeight` method to consider `undefined` as an invalid value